step/src/main/java/com/google/sps/DatabaseHandler.java

```diff
@@ -51,6 +51,15 @@ private DatabaseHandler() {
   public static void addUser(String firstName, String lastName, int dayBirth,
     int monthBirth, int yearBirth, String email, String id) {
     Entity entity = new Entity("User", id);
+
+    if (firstName == null) {
+      firstName = "Unknown";
+    } 
+    
+    if (lastName == null) {
+      lastName = "Unknown";
```

Comment:
Can we change this string to a constant?

Reply:
+1